packages/block-library/src/navigation/edit/navigation-menu-delete-control.js

```diff
@@ -19,8 +19,7 @@ export default function NavigationMenuDeleteControl( { onDelete } ) {
 	return (
 		<>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
This can be seen, while adding the navigation menu, going to inspector controls advanced tab, and check for `Delete Menu` button.

| Before | After     |
| :-------- | :------- |
| <img width="357" alt="navigation-delete-before" src="https://github.com/user-attachments/assets/f776680a-2725-44da-850c-9a37e263f1f8"> | <img width="357" alt="navigation-delete-after" src="https://github.com/user-attachments/assets/43544e1d-081e-4359-843e-a0c6859865a6"> |